src/lib/telldus-accessory.js

```diff
@@ -47,6 +48,12 @@ class TelldusAccessory {
     this.log = string => log(`[${this.name}]: ${string}`)
     this.Characteristic = homebridge.hap.Characteristic
     this.Service        = homebridge.hap.Service
+
+    // Support age limit for sensors to avoid stale information.
+    this.isStale = sensor => {
```

Comment:
I'd rather have this implemented as a method in the class:
```js
class TelldusAccessory {
  constructor(/*...*/) {/*...*/}
  
  isStale(sensor) {
    //...
  }
}
```
than attach the method to `this`, to conform with how the rest of the code is implemented.

Another thought about this; when would you need to worry about a sensor being passed to this method is being undefined?

_The way the `log`-function is declared is a remnant from an earlier version, and might be solved better in another PR, using a similar implementation._